lib/inertia_rails/controller.rb

```diff
@@ -17,5 +24,13 @@ def inertia_location(url)
       headers['X-Inertia-Location'] = url
       head :conflict
     end
+
+    def inertia_redirect_to(options = {}, response_options = {})
+      if (errors = response_options.delete(:errors))
+        session[:inertia_errors] = errors
```

Comment:
in the session, we use the `:inertia_errors` key so that we (hopefully) don't collide with the including application's session variables

Reply:
What is the collision risk here? It looks like we're deleting the session errors anyway so why not use that location?